### PR TITLE
Add logging.

### PR DESCRIPTION
**Pull request by tomusdrw**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -16,7 +17,13 @@ const transport = new Parity.Api.Transport.Http(`http://localhost:${config.get('
 const api = new Parity.Api(transport);
 
 const app = express();
+// Support health checking by sending HEAD
+app.head('/', (req, res) => res.status(200).end());
```
Comment (ngotchac) [inline server.js]: Maybe also the Parity node status ? Like block number/timestamp... ? (not in a HEAD request though)

File: package.json [hunk 2]
```diff
@@ -17,6 +17,8 @@
     "config": "1.29.4",
     "express": "^4.13.3",
     "js-sha3": "^0.5.5",
+    "morgan": "1.9.0",
```
Comment (ngotchac) [inline package.json]: Is this necessary?

Comment (tomusdrw) [reply package.json]: Good catch!

File: server.js (hunk 1 above)
Comment (tomusdrw) [reply server.js]: Makes sense. Added `GET /health` endpoint.

**Review by ngotchac — APPROVED** [review]

**Pull request merged**